tutorials/notebooks/pytorch/export/example_pytorch_export.ipynb

```diff
@@ -43,6 +43,35 @@
       "execution_count": null,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "In order to export your quantized model to onnx format, and use it for inference, some additional packages are needed. Notice, this is needed only for models exported to onnx, so this part can be skipped if this is not planned:"
```

Comment:
Consider adding a warning that running inference on the ONNX exported model will take a very long time (maybe also explain why and why the additional package is needed? if you think it is relevant for the user)